qiskit_ibm_experiment/VERSION.txt

```diff
@@ -1 +1 @@
-0.2.4
+0.2.5
```

Comment:
Are developers expected to be aware of the necessity to change here?
Is there something that does it automatically?
Or at least that blocks merging if it's not done?

Reply:
`qiskit-experiments` is dependent on a specific version of `qiskit-ibm-experiment`. By raising the version here and there, we ensure in experiments we work with a bug-free version.